src/ert/scheduler/slurm_driver.py

```diff
@@ -73,8 +73,8 @@ def __init__(
         realization_memory: Optional[int] = 0,
         queue_name: Optional[str] = None,
         memory_per_cpu: Optional[str] = None,
-        max_runtime: Optional[str] = None,
-        squeue_timeout: Optional[str] = None,
+        max_runtime: Optional[float] = None,
+        squeue_timeout: float = 2,
```

Comment:
Is this value ever to be changed? ie. why to expose this to the users?

Reply:
We keep it only for backwards compatibility.